aten/src/ATen/functorch/BatchRulesConvolution.cpp

```diff
@@ -138,81 +138,6 @@ static Tensor _convolution_decomp(
       input_r, weight_r, bias_r_opt, stride_, padding_, dilation_, transposed_, output_padding_, groups_);
 }
 
-// TODO: delete the following after confirming performance
```

Comment:
I presume we've confirmed performance by now :P

Reply:
lol yes